doc/cephadm/services/index.rst

```diff
@@ -357,7 +357,8 @@ Or in YAML:
 Placement by pattern matching
 -----------------------------
 
-Daemons can be placed on hosts as well:
+Daemons can be placed on hosts using a host pattern as well.
+By default, the host pattern is matched using fnmatch:
```

Comment:
Unrelated to this PR directly, so no need to change anything but: if the reader is not already familiar with python it may not be clear what `fnmatch` is. Maybe a follow up change linking to the fnmatch docs might be enough.

Reply:
I wondered myself ;)

Reply:
Added an fnmatch link and mentioned it supports UNIX shell-style wildcards